pkgs/development/libraries/libelf/default.nix

```
@@ -27,6 +27,11 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
 
+  preConfigure = lib.optionalString (stdenv.hostPlatform.useAndroidPrebuilt) ''
```

Can just do
```suggestion
  preConfigure = if !stdenv.hostPlatform.useAndroidPrebuilt then null else ''
```
and mass rebuild separately.